addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -633,6 +633,10 @@ export class RelationalModel extends Model {
         let data;
         if (reload) {
             data = await this._loadData(tmpConfig);
+            if (data.records && !data.records.length && tmpConfig.offset) {
```

Comment:
```suggestion
            if (data.records?.length === 0 && tmpConfig.offset) {
```
can you please check it?

Reply:
Hello sir,
Thanks for the suggestion. I have implemented it and updated the pr.